hw/ip/aes/doc/_index.md

```diff
@@ -315,7 +319,7 @@ The key length is configured using the KEY_LEN field of {{< regref "CTRL" >}}.
 Independent of the selected key length, software must always write all 8 32-bit registers.
 Each register must be written at least once.
 The order in which these registers are written does not matter.
-Anything can be written to the unused key registers.
+Anything can be written to the unused key registers, but random data is written to them ideally.
```

Comment:
> Anything can be written to the unused key registers, but random data is written to them ideally.

Maybe?:
"Anything can be written to the unused key registers, however, random data is preferred."

Reply:
Sounds good.